specs/Media/MediaEvent/example.json

```diff
@@ -0,0 +1,16 @@
+{
+  "id": "mediaEvent_1509702324600",
+  "type": "MediaEvent",
+  "eventType": "plate-detected",
+  "mediasource": {
```

Comment:
not consistent with the example shown in the spec. Not consistent with the spec.

Reply:
fixed example